x-pack/plugins/canvas/canvas_plugin_src/functions/server/src/esdocs.js

```diff
@@ -101,6 +101,16 @@ export const esdocs = () => ({
           columns,
           rows,
         };
+      })
+      .catch(e => {
```

Comment:
Just a note, there's a ton of code duplication between `essql` and `esdocs` that would be nice to unify.